src/modules/organisation-role/organisation-role.service.ts

```diff
@@ -124,4 +129,36 @@ export class OrganisationRoleService {
       throw new Error(`Failed to fetch role: ${error.message}`);
     }
   }
+
+  async deleteRole(organisationId: string, roleId: string, currentUser) {
+    if (!['superadmin', 'admin', 'owner'].includes(currentUser.role)) {
+      throw new UnauthorizedException('You are not authorized to manage roles');
+    }
```

Comment:
What is line 134 meant to be doing. Please explain 🙏🏽

Reply:
You mean 135, it's mean to check the permission of the person that want's to delete the role, but I don't think it's important. 

I will remove it